backend/prisma/schema.prisma

```diff
@@ -118,23 +118,22 @@ model Contact {
   owner              User                      @relation(fields: [ownerID], references: [id])
   importantDateEvent ImportantDateEvent[]
   relationships      UserContactRelationship[]
-  Plan               Plan?                     @relation(fields: [planId], references: [id])
-  planId             String?
+  Plan               Plan[]
 }
 
 model ImportantDateEvent {
-  id        String  @id @default(uuid())
-  contact   Contact @relation(fields: [contactID], references: [id])
+  id        String    @id @default(uuid())
+  contact   Contact   @relation(fields: [contactID], references: [id])
   contactID String
   date      String
-  eventType String
+  eventType EventType @default(OTHER)
 }
 
 model UserContactRelationship {
-  id               String  @id @default(uuid())
-  contact          Contact @relation(fields: [contactID], references: [id])
+  id               String           @id @default(uuid())
+  contact          Contact          @relation(fields: [contactID], references: [id])
   contactID        String
-  relationshipType String
+  relationshipType RelationshipType @default(OTHER)
```

Comment:
Looks good!

Reply:
Thank you sir!

Reply:
Decided to not do the enums for this. Might come back and do this enum transformation later on but not a priority rn